torch/quantization/utils.py

```diff
@@ -133,3 +134,49 @@ def check_min_max_valid(min_val: torch.Tensor, max_val: torch.Tensor) -> bool:
         ), "min {} should be less than max {}".format(min_val, max_val)
 
     return True
+
+
+def calculate_qmin_qmax(quant_min: int, quant_max: int, has_customized_qrange: bool, dtype: torch.dtype,
```

Comment:
Why is the _ removed? Do we see this as a user facing function?

Reply:
Kept it consistent with the rest of functions in this file. Since it is in utils it shouldn't be a user facing function.